requirements/flask20.in

```diff
@@ -0,0 +1,5 @@
+authlib==1.2.1
```

Comment:
The purpose of the flask20 test is to test the minimum versions of the install requirements, so this file should include the following definition:

```
# minimum install requirements
flask==2.0.0
flask-marshmallow==0.12.0
webargs==8.3.0
flask-httpauth==4.0.0
apispec==6.0.0

# for example applications
flask-sqlalchemy==2.5.1
sqlalchemy==1.4.49
marshmallow-dataclass==8.5.0
authlib==1.2.1
```

And the name `min` (for test env, requirement file, etc.) would be more appropriate.